bindings/nim/src/CouchbaseLite/private/fl.nim

```diff
@@ -10,7 +10,12 @@
 ##         Fleece commit 4ca3dbf1, "Mutable.hh: Allow conversion of keyRef and FLSlot to FLSlot".
 
 
-{.push dynlib: "libCouchbaseLiteC.dylib".}
+when defined(Linux):
+  {.push dynlib: "libCouchbaseLiteC.so".}
+elif defined(MacOS) or defined(MacOSX):
+  {.push dynlib: "libCouchbaseLiteC.dylib".}
+elif defined(Windows):
+  {.push dynlib: "libCouchbaseLiteC.dll".}
```

Comment:
I think this would be CouchbaseLiteC.dll, not libCouchbaseLiteC.dll (or does Nim do something about this behind the scenes?)

Reply:
Ah, that's probably the case. Wasn't sure what naming scheme you used so I just guessed.